Doc/library/os.rst

```diff
@@ -1092,6 +1092,64 @@ or `the MSDN <https://msdn.microsoft.com/en-us/library/z0kc8e3z.aspx>`_ on Windo
    .. versionadded:: 3.3
 
 
+.. function:: preadv(fd, buffers, offset, flags=0)
+
+   Read from a file descriptor *fd* from offset *offset* into mutable
+   :term:`bytes-like objects <bytes-like object>` *buffers*. Transfer data into
+   each buffer until it is full and then move on to the next buffer in the
+   sequence to hold the rest of the data.
+
+   Return the total number of bytes read which can be less than the total
+   capacity of all the objects.
+
+   Combine the functionality of :func:`os.readv` and :func:`os.pread`.
+
+   The flags argument contains a bitwise OR of zero or more of the following
+   flags:
+
+   - :data:`RWF_HIPRI`
+   - :data:`RWF_NOWAIT`
+
+   Using non-zero flags requires Linux 4.6 or newer.
+
+   The operating system may set a limit (:func:`sysconf` value
+   ``'SC_IOV_MAX'``) on the number of buffers that can be used.
+
+   Availability: Linux 2.6.30 abd newer, FreeBSD 6.0 and newer,
```

Comment:
should be "Availability: Linux 2.6.30, FreeBSD 6.0 and newer."

Reply:
"and newer" is commonly used in os.rst documentation (i fixed the typo)